### DISCUSSION TITLE
Is the package still maintained?

### DISCUSSION THREAD

**Discussion by valentinhttr** [Cosium/spring-data-jpa-entity-graph#218] [General]
Hello!

I came across your package, surprised that it's not implemented in data-jpa core. Especially the use of @EntityGraph on specifications.
So I want to use your package, but I'd like to make sure it's still maintained?

Thanks

**Comment by reda-alaoui** [comment]
Yep, still kicking

**Reply by valentinhttr** [reply]
Wow, that was fast! Thanks for your feedback and once again for your package.
If I have the opportunity to PR or support you in any way, I will! Thank you
